extra_model/_models.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Optional, Tuple, Union
+
+
+class ModelBase:
+    def load_from_files(self):
```

Comment:
is this the only function that we need to have? What about `predict` and `train`?

Reply:
I just pushed a change that should better explain the intention of the two classes.

`ModelBase` is meant as a stub to be replaced with acme_modelyzer.Model when used at Wayfair, via the factory function. A goal of the project is to use the open source package at Wayfair when it is published and to stop using / development of the internal package. So we need a way to swap out the base class so we can continue to use acme_modelyzer.Model at Wayfair.

`ExtraModelBase` is meant as a container for ExtraModelV0 functionality, but it will never be instantiated directly.

Here is the content from my JIRA ticket:

From my investigation, `model.load_files` is the only method required from the AcmeModelyzer base model, so I have stubbed that out in a base class local to ExtraModel. I setup a factory pattern that allows for injecting a base class other than the default when ExtraModel is created. This allows for adoption at Wayfair via the follwing pattern:

```python
from extra_model import model_factory
from acme_modelyzer.models import Model as AcmeModel

ExtraModel = extra_factory(AcmeModel)
```

The default pattern:
```python
from extra_model import model_factory

ExtraModel = extra_factory()
```

altough this may be made simpler (just an import) if I see the opportunity